### ISSUE TITLE
Digital Measure Import tripping up

### ISSUE THREAD

**Issue by DanCoughlin** [psu-stewardship/researcher-metadata#188]
Most import errors prevent the imports from completing - with the assumption right now being that we'll see the error, fix the problem right away, and then the import will begin to succeed again. Probably the best that we can do is really broadly capture the errors, log information about them somewhere, and try to notify someone. That's already happening, but it's really only West Arete that has total visibility into the errors and it's only West Arete that gets any kind of notification.

We need to update visibility into the errors for Penn State, and have Penn State receive notification of the errors on this import.

**Comment by rschenk** [comment]
> log information about them somewhere, and try to notify someone

This is too nebulous for me to implement, I don't know what tools PSU wants me to use. Where should these errors  be logged and how and who?

**Comment by EricDurante** [comment]
@rschenk Yes, this is nebulous. We have a bunch of different automated data imports that run all of the time. This is just one of them. Sometimes these imports encounter an intermittent error and die, but then they succeed the next time they run. Other times they begin to encounter the same error on every run, and so they never complete successfully from that point on.

What we care most about is making sure that none of the imports are ever permanently failing. Logging error info is important but secondary, and notification is tertiary, I would say.

For now, my plan for this (and other import errors) was to catch any errors raised by the import processes that aren't otherwise handled and just log the error info in the database where all RMD admins and developers can easily see it. Sort of like how I'm already doing with [email errors](https://github.com/psu-stewardship/researcher-metadata/blob/807ceeaeae231812ff07417917aff20b7bda3150/app/models/open_access_notifier.rb#L42). Only this would be something like import_errors and it would track the type of import (Activity Insight in this case), the type of error, and other error info/stacktrace/etc. 

That's my thought anyway. We could also continue to send error info to Bugsnag and look at sharing access to the Bugsnag project. Or if Bugsnag works well enough then we could do that instead of logging in the DB. Again, the key thing here is to make sure that the imports can proceed in the face of one-off errors regardless of how we log/notify about the errors. 

Bugsnag is the place to look currently for any unhandled errors that are being raised by imports (rake tasks).
